tests/arxiv/run_large_scale_test.sh

```
@@ -0,0 +1,321 @@
+    print(f"    G (Grounding):        {G:.3f} (weight: {wG:.2f})")
+    print(f"    Ctx (Combined):       {Ctx:.3f}")
+    print(f"    α (Exponent):         {alpha:.2f}")
+    print(f"  ─────────────────────────────")
+    print(f"  Conveyance Score (C): {C:.4f}")
+    print(f"  Formula: C = (W·R·H/T)·Ctx^α")
+
+except Exception as e:
+    print(f"  Error computing Conveyance Score: {e}")
+    print(f"  Using default values for demonstration")
+EOF
+fi
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)